### PR TITLE
[New] Django with One-Click Apps

### PR DESCRIPTION
**Pull request by hzoppetti**
draft guide

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit e2d9b6fd7af31d6d6ba444fbd57a12b4f8990d1f

https://deploy-preview-3192--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Django Options
+
+You can configure your Django App by providing values for the following fields:
+
+| **Field** | **Description** |
+|:--------------|:------------|
+| **Django APP Name** | Name for the Django application. This field cannot have spaces. *Required*. |
+| **Django USER** | Username for your Django application. *Required*. |
+| **Django Password** | Password for the Django USER. *Required*. |
+| **Django USER email** | Email address for the Django USER. The format for the email must be in the form user@email.tld *Required*. |
```
Comment (andystevensname) [inline docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: Period after "user@email.tld"

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+| **Linode Label** | The name for your Linode, which must be unique between all of the Linodes on your account. This name will be how you identify your server in the Cloud Manager’s Dashboard. *Required*. |
+| **Root Password** | The primary administrative password for your Linode instance. This password must be provided when you log in to your Linode via SSH. It must be at least 6 characters long and contain characters from two of the following categories: lowercase and uppercase case letters, numbers, and punctuation characters. Your root password can be used to perform any action on your server, so make it long, complex, and unique. *Required*. |
+
+When you've provided all required Linode Options, click on the **Create** button. **Your Django app will complete installation anywhere between 2-5 minutes after your Linode has finished provisioning**.
+
+## Getting Started after Deployment
+
+### Access Django
+
+After Django has finished installing, you will be able to access your Django site over `https://yourlinodeip:8000` with your Linode's IPv4 address.
```
Comment (andystevensname) [inline docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: "...access your Django site at your Linode's IPv4 address, for instance: https://yourlinodeIP:8000."

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+1.  You will be able to access your Django site by copying your Linode's IPv4 address and entering it in the browser of your choice. To find your Linode's IPv4 address. Click on the **Linodes** link in the sidebar. You will see a list of all your Linodes.
```
Comment (andystevensname) [inline docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: Probably don't need the first sentence here.

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md (hunk 1 above)
Comment (hzoppetti) [reply docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: updated.

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md (hunk 2 above)
Comment (hzoppetti) [reply docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: updated

File: docs/platform/one-click/deploying-django-with-one-click-apps/index.md (hunk 3 above)
Comment (hzoppetti) [reply docs/platform/one-click/deploying-django-with-one-click-apps/index.md]: removed

**Pull request merged**